.github/workflows/build.yml

```diff
@@ -31,6 +31,44 @@ jobs:
           pip install tox tox-gh-actions
     - name: Run pre-tests with Tox
       run: tox -e ${{ matrix.toxenv }}
+
+  test-dashboard-build:
```

Comment:
This should be added into the list of `needs` of PyPI and conda below.

Reply:
Ok!